addon/ng2/models/webpack-build-common.ts

```diff
@@ -0,0 +1,92 @@
+import * as webpack from 'webpack';
+import {LoaderConfig} from '../utilities/ts-path-mappings-webpack-plugin';
+
+const path = require('path');
+const ForkCheckerPlugin = require('awesome-typescript-loader').ForkCheckerPlugin;
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+export const getWebpackCommonConfig = function(projectRoot: string) {
```

Comment:
Also, return a type; Please use the `@types/webpack` package or install the typings for it from definitely typed. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/webpack/webpack.d.ts